samples/README.md

```diff
@@ -0,0 +1,13 @@
+# Job Sample Overview
+
+This sample demonstrates how to start jobs using your own script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites    
+
+This sample assumes that you have already configured an Kubernetes cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do).
```

Comment:
Can we be more specific - installed CRDs and controller instead of saying just operator here.